.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
```

Comment:
I have thousands of notifications running through my GitHub daily, and I will most definitely dismiss these as noise, therefore adding this would become quite useless.